packages/client/src/object/convertWireToOsdkObjects/createOsdkInterface.ts

```diff
@@ -50,7 +50,7 @@ export function createOsdkInterface<
   };
 
   if (process.env.TARGET !== "browser") {
-    Object.setPrototypeOf(interfaceHolder, OsdkCustomInspectPrototype);
+    Object.setPrototypeOf(interfaceHolder, null);
```

Comment:
So this is what we discussed, are we still good with this for now?

Reply:
Lets revert this change for now. It only makes sense to do this in non-browser. If it rears its head again, we can come up with a better long term solution.